src/main/java/io/qmbot/telegrambot/AboutCommand.java

```diff
@@ -0,0 +1,37 @@
+package io.qmbot.telegrambot;
+
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Chat;
+import org.telegram.telegrambots.meta.api.objects.Message;
+import org.telegram.telegrambots.meta.api.objects.User;
+import org.telegram.telegrambots.meta.bots.AbsSender;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+public class AboutCommand extends BotCommand {
+    public AboutCommand() {
+        super("about", "Display information about the bot");
+    }
+
+    public void execute(AbsSender absSender, User user, Chat chat, String[] strings) {
+        SendMessage message = new SendMessage();
+        message.setChatId(chat.getId());
+        message.setText(getDescription());
+        try {
+            absSender.execute(message);
+        } catch (TelegramApiException e) {
```

Comment:
pls replace "e.printStackTrace" with exception logging here and in other places in this change